Dockerfile

```diff
@@ -0,0 +1,35 @@
+FROM mhart/alpine-node:11
+
+MAINTAINER samparsky@gmail.com
+
+ARG ARG_PORT=8005
+ARG ARG_ADAPTER=ethereum
+ARG ARG_IDENTITY=awesomeLeader
```

Comment:
this arg and generally all args related ot the dummy adapter should not be there

Reply:
done